platform/podjs/src/browserPod.ts

```diff
@@ -55,60 +55,56 @@ async function openDatabase(): Promise<IDBDatabase> {
     });
 }
 
-/**
- * It implements the `PolyIn` interface by storing quads using Oxigraph
- * and syncing the storage to IndexedDB
- * @class IDBPolyIn
- */
-class OxigraphPolyIn implements PolyIn {
-    private store: Promise<oxigraph.Store> = this.init();
-    private pendingSync: Promise<void> | null = null;
-
-    private async init(): Promise<oxigraph.Store> {
-        type wasmModule = () => Promise<WebAssembly.Module>;
-        const [, data] = (await Promise.all([
+const oxigraphStore = (async () => {
+    type wasmModule = () => Promise<WebAssembly.Module>;
+    const data = (
+        await Promise.all([
             initOxigraph((oxigraphWasmModule as unknown as wasmModule)()),
             openDatabase().then(
                 (db) =>
                     new Promise((resolve, reject) => {
-                        const tx = db.transaction(
-                            [OBJECT_STORE_POLY_IN],
-                            "readonly"
-                        );
+                        const objectStores = [OBJECT_STORE_POLY_IN];
```

Comment:
Maybe change the name to something generic like OBJECT_STORE_OXIGRAPH?

Reply:
Good idea. Renamed it to `OBJECT_STORE_QUADS` since the data we store in there aren't specific to Oxigraph, and can be parsed or generated with any software that supports the `n-quads` format. What do you think?

Reply:
Sorry I agree :D didnt see